trainer/lib/trainer/test_parser.rb

```diff
@@ -191,9 +215,12 @@ def summaries_to_data(summaries, failures)
       rows = testable_summaries.map do |testable_summary|
         all_tests = testable_summary.all_tests.flatten
 
+        tests_by_identifier = {}
```

Comment:
Ah, I should comment this! I can't run a unique on that because I still need all the test rows. This is specifically for Xcode 13's retry logic. If a test fails and retries, it will appear multiple times. 

So this is used to find the last run of it to see if it passed. But... some people will still want the total count of tests and to see the number of retries. I should rename this but I should also maybe find a way to capture how many times each individual tests were run ( because right now it doesn't do that) 🤔

Reply:
Ah that's right. Thanks!